File: templates/lang/android_app_e6/src_files/main/MainActivity.java
package {{ .Group }}.main;

import android.os.Bundle;

import {{ .Group }}.R;
import {{ .Group }}.base.BaseAppCompatActivity;

public class MainActivity extends BaseAppCompatActivity {

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);

    if (getFragmentManager().findFragmentById(R.id.main_content) == null) {
      getFragmentManager().beginTransaction()
          .add(R.id.main_content, new MainFragment())
          .commit();
    }
  }
}

File: templates/lang/kotlin_android_app_dagger/src_files/main/MainActivity.kt
package {{ .Group }}.main

import android.os.Bundle
import {{ .Group }}.R
import {{ .Group }}.base.BaseAppCompatActivity

class MainActivity : BaseAppCompatActivity() {

  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.activity_main)

    if (supportFragmentManager.findFragmentById(R.id.main_content) == null) {
      supportFragmentManager.beginTransaction()
          .add(R.id.main_content, MainFragment())
          .commit()
    }
  }
}
